### PR TITLE
LPS-126648 | Master

### PR DESCRIPTION
**Pull request by CalumR23**
[LPS-126648](https://issues.liferay.com/browse/LPS-126648)
QA Review Needed
@david-gutierrez-mesa 
@Tim-Cao 
@liulukiwi

### CODE REVIEW

**Comment by CalumR23** [comment]
ci:test:relevant

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
ci:test:relevant

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
ci:test:revelant

**Comment by nikki-pru** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
ci:test:relevant

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverter.testcase [hunk 1]
```diff
@@ -855,7 +855,24 @@ definition {
...
+		SitePages.openPagesAdmin(siteURLKey = "Test Site Name");
+
+		SitePages.gotoPageMenuItem(
+			menuItem = "Convert to content page...",
+			pageName = "Test Page Name");
+
+		PageEditor.addWidget(
+			portletName = "Web Content Display",
+			targetFragmentName = "Asset Publisher");
 	}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverter.testcase:876]: Add additional steps to assert the web content is shown in converted content page in view mode.

**Comment by CalumR23** [comment]
Added the changes @Tim-Cao described above, please review Pull Request @david-gutierrez-mesa @liulukiwi.

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
ci:test:relevant

**Comment by CalumR23** [comment]
Accidentally added new commits from master, will wait for merge.

**Comment by CalumR23** [comment]
Failures are not related to the pull.

**Comment by david-gutierrez-mesa** [comment]
@CalumR23 can you please rebase and resend with your commits. Thanks!

**Comment by CalumR23** [comment]
I have rebased with master and am ready for review. @Tim-Cao  @david-gutierrez-mesa @liulukiwi

**Comment by CalumR23** [comment]
ci:test:sf

**Comment by CalumR23** [comment]
ci:test:relevant

**Comment by Tim-Cao** [comment]
Hi @CalumR23, the verb should be always present tense in commit message.

**Comment by CalumR23** [comment]
Hey @Tim-Cao, I have changed my commit message.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverter.testcase [hunk 2]
```diff
@@ -855,7 +855,24 @@ definition {
...
-		// TODO LPS-125933 ConvertWidgetPageWithWebContentsToContentPage pending implementation
+		JSONLayout.addPublicLayout(
+			groupName = "Test Site Name",
+			layoutName = "Test Page Name");
+
+		JSONLayout.addWidgetToPublicLayout(
+			groupName = "Test Site Name",
+			layoutName = "Test Page Name",
+			widgetName = "Asset Publisher");
+
```
[the remark this answers is not in the document]
Comment (CalumR23) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverter.testcase:901]: Lines 902-905 fix thus

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverter.testcase [hunk 3]
```diff
@@ -855,7 +855,24 @@ definition {
...
+		SitePages.openPagesAdmin(siteURLKey = "Test Site Name");
+
+		SitePages.gotoPageMenuItem(
+			menuItem = "Convert to content page...",
+			pageName = "Test Page Name");
+
```
[the remark this answers is not in the document]
Comment (CalumR23) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverter.testcase:912]: Line 919 fixes this

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverter.testcase (hunk 1 above)
Comment (CalumR23) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/pagesadmin/ContentPageConverter.testcase:920]: Line 919 fixes this

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/3545
View total diff https://github.com/liferay-echo/liferay-portal/pull/3545/commits/570ea1b16f95b41553debfd23420bef6128d92ca

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**